DonjinKrawler-Console/src/main/java/donjinkrawler/Console.java

```diff
@@ -0,0 +1,85 @@
+package donjinkrawler;
+
+import com.esotericsoftware.kryonet.Client;
+import com.esotericsoftware.kryonet.Connection;
+import com.esotericsoftware.kryonet.Listener;
+import donjinkrawler.interpreter.InvalidCommandException;
+import donjinkrawler.interpreter.Expression;
+import krawlercommon.PlayerData;
+import krawlercommon.RegistrationManager;
+import krawlercommon.packets.GetPlayersPacket;
+import krawlercommon.packets.LoginPacket;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+import java.util.Scanner;
+
+public class Console {
```

Comment:
Reiktu final pridėti.

[CHECKLIST](https://dzone.com/articles/java-code-review-checklist):
`Make class final if not being used for inheritance.`